enterprise/admin.py

```diff
@@ -43,6 +53,11 @@ class EnterpriseCustomerAdmin(SimpleHistoryAdmin):
     class Meta(object):
         model = EnterpriseCustomer
 
+    @staticmethod
+    def logo(instance):
+        # instance is Enterprise customer
+        return instance.branding_configuration.logo
```

Comment:
@asadiqbal08 Unless I'm missing something this should crash the view when no branding configuration is created for the EC. Should it include `instance.branding_configuration is not None` check?